packages/jobs/lib/execution/sync.ts

```diff
@@ -70,7 +70,7 @@ export async function startSync(task: TaskSync, startScriptFn = startScript): Pr
         team = accountAndEnv.account;
         environment = accountAndEnv.environment;
 
-        syncType = syncConfig.sync_type === SyncType.INCREMENTAL && lastSyncDate ? SyncType.INCREMENTAL : SyncType.FULL;
+        syncType = syncConfig.sync_type?.toLowerCase() === SyncType.INCREMENTAL.toLowerCase() && lastSyncDate ? SyncType.INCREMENTAL : SyncType.FULL;
```

Comment:
Sync config sync_type is now lowercase https://github.com/NangoHQ/nango/blob/7c376b44ec2775c86da11bb281b941df8c49c54f/packages/types/lib/nangoYaml/index.ts#L2 We should probably fix this but I don't want to introduce any regression until I checked with @bodinsamuel For now just fixing the SyncJob.